.gitignore

```diff
@@ -7,3 +7,8 @@ inventory/host_vars/*/secrets.yml
 *.log
 **/*.retry
 .idea/
+
+secrets/*
```

Comment:
Are there some additional secrets files here that are needed to use it, and if so where will we store them?

Reply:
See the documentation :point_right: https://github.com/openfoodfoundation/ofn-install/wiki/UK%27s-split-infrastructure